pax/simulation.py

```diff
@@ -465,33 +468,31 @@ def make_pax_event(self):
 
             # Did you want to superpose onto real noise samples?
             if self.config['real_noise_file']:
-                if noise_sample_mode != 'coherent':
+                #if noise_sample_mode != 'coherent':
                     # For each channel, choose different noise sample numbers
-                    chosen_noise_sample_numbers = np.random.randint(0,
+                    #chosen_noise_sample_numbers = np.random.randint(0,
                                                                     available_noise_samples - 1,
                                                                     needed_noise_samples)
 
-                    roll_number = np.random.randint(noise_sample_len)
+                    #roll_number = np.random.randint(noise_sample_len)
 
                 # Extract the chosen noise samples and concatenate them
                 # Have to use a listcomp here, unless you know a way to select multiple slices in numpy?
                 #  -- yeah making an index list with np.arange would work, but honestly??
-                real_noise = np.concatenate([
-                    self.noise_data[channel - self.channel_offset][nsn * noise_sample_len:(nsn + 1) * noise_sample_len]
-                    for nsn in chosen_noise_sample_numbers
-                ])
+
+                real_noise = noise_to_add[channel-self.channel_offset]
 
                 # Roll the noise samples by a fraction of the sample size,
                 # to avoid same artifacts falling at the same point every time
-                np.roll(real_noise, roll_number, axis=0)
+                #np.roll(real_noise, roll_number, axis=0)
```

Comment:
__[flake8]__
* [E122] continuation line missing indentation or outdented
* [E265] block comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->